src/v2/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebar.jest.tsx

```diff
@@ -36,4 +65,33 @@ describe("ArtworkSidebar", () => {
     const wrapper = getWrapper()
     expect(wrapper.find(ArtworkSidebarMetadata).length).toBe(1)
   })
+
+  it("renders the create alert section when artwork-page-create-alert ff is enabled", () => {
+    mockFeatureFlags = {
+      featureFlags: {
+        "artwork-page-create-alert": {
+          flagEnabled: true,
+          variant: { name: "enabled", enabled: false },
+        },
+      },
+    }
+
+    mockUseSystemContext.mockImplementationOnce(() => mockFeatureFlags)
+
+    renderWithRelay()
+
+    expect(screen.queryByText(/Create Alert/i)).toBeInTheDocument()
+    expect(
+      screen.queryByText(/Be notified when a similar piece is available/i)
+    ).toBeInTheDocument()
+  })
+
+  it("does not render the create alert section when artwork-page-create-alert ff is enabled", () => {
```

Comment:
Is the spec description ("… when artwork-page-create-alert ff is enabled") correct for this one? Or should it be "disabled"?

Reply:
good catch ;) will fix that in a follow up pr